newsletter-january-2023.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023! In this newsletter, we present the updates happening in 2022's last month and since the :doc:`previous newsletter </newsletter-december-2022>`.
+
+.. Commenting out this stuff, isn't it better that 
+.. 
+.. News and updates
+.. ----------------
+.. 
+.. Here are the latest updates from our team:
+
+- 📹️ Eric's talk at DjangoCon US can be watched here: `Documenting Django Code in 2022`_
+- 🛠️ We are busy reorganizing our own documentation. Since last newsletter, we have opened or merged 20 documentation PRs. We look forward to a "show and tell" about how all of this was refactored using Diátaxis framework.
+- 🚢️ Sphinx 6.0 and 6.1 have been released. Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- ⚙️ Add new stuff here
+- 💡️ Choose your own emoji
+- 🔍️ 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
```

Comment:
We should probably mention https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-368m-86q9-m99w

Reply:
Added :+1: 

I hope it's fine to add with no further context, I feel like it's hard to give a quick summary and that only people that self-host really need to review it.